--- origsrc/Apetag/basic.C	2007-11-03 12:34:22.000000000 -0500
+++ src/Apetag/basic.C	2008-07-06 01:52:47.343750000 -0500
@@ -44,7 +44,9 @@
 #include <fstream>
 
 // for backtrace
+#ifdef __linux__
 #include <execinfo.h>
+#endif
 
 // ========================================================================
 
--- origsrc/Apetag/basic.H	2007-12-16 11:08:10.000000000 -0600
+++ src/Apetag/basic.H	2008-07-06 01:59:42.046875000 -0500
@@ -71,6 +71,7 @@
 
 // ========================================================================
 
+inline string decstr(int val, UINT32 width=0 )    {return StringDecSigned((int)val,width);}
 inline string decstr(INT32 val, UINT32 width=0 )  {return StringDecSigned((INT32)val,width);}
 inline string decstr(INT16 val, UINT32 width=0 )  {return StringDecSigned((INT32)val,width);}
 
